devices/fields/usb/v1.yaml

```diff
@@ -137,6 +137,9 @@ DADDR:
 BTABLE:
   BTABLE: [0, 0b1111111111111]
 EP*R:
+  _modify:
+    STAT_[RT]X:
+      access: read-write
```

Comment:
Just out of curiosity, this bit is marked as `t` for `toggle` in [RM0440](https://www.st.com/resource/en/reference_manual/rm0440-stm32g4-series-advanced-armbased-32bit-mcus-stmicroelectronics.pdf). Is that something we care about?

> Bits 13:12 STAT_RX [1:0]: Status bits, for reception transfers
> [...] These bits can be toggled by software to initialize
their value. When the application software writes ‘0, the value remains unchanged, while
writing ‘1 makes the bit value toggle.

Reply:
There is `_W1T` in line 153.

Reply:
Oh, ok. Interesting. Thanks :)